Sources/TuistAutomation/Utilities/TargetBuilder.swift

```diff
@@ -1,4 +1,5 @@
 import TSCBasic
+import struct TSCUtility.Version
```

Comment:
We usually do `import TSCUtility` and then use `TSCUtility.Version` below in case of name clashes

Reply:
Done!